test/parallel/test-cluster-inspector-debug-port.js

```diff
@@ -27,6 +27,8 @@ if (cluster.isMaster) {
   fork(4, ['--inspect', '--debug']);
   fork(5, [`--debug=${debuggerPort}`, '--inspect']);
   fork(6, ['--inspect', `--debug-port=${debuggerPort}`]);
+  fork(7, [`--inspect-port=${debuggerPort}`]);
+  fork(8, ['--inspect', `--inspect-port=${debuggerPort}`]);
```

Comment:
As @cjihrig mentions, this test isn't setup to be able to handle `brk` flag. It already omits `--debug-brk`.

I do think it would be good to add some testing to make sure `cluster` handling of `brk` options doesn't regress, so I opened a new issues for that: https://github.com/nodejs/node/issues/11420.